src/appkit.cs

```diff
@@ -23589,4 +23605,15 @@ interface NSFilePromiseReceiver : NSPasteboardReading
 		[Export ("receivePromisedFilesAtDestination:options:operationQueue:reader:")]
 		void ReceivePromisedFiles (NSUrl destinationDir, NSDictionary options, NSOperationQueue operationQueue, Action<NSUrl, NSError> reader);
 	}
+
+	//TODO - Waiting on CloudKit binding
+	//[Protocol]
+	//[Mac (10,12)]
+	//interface NSCloudSharingValidation
+	//{
+	//	[Abstract]
+	//	[Export ("cloudShareForUserInterfaceItem:")]
+	//	[return: NullAllowed]
+	//	CKShare CloudShareForUserInterfaceItem (NSValidatedUserInterfaceItem item);
```

Comment:
Yep. Was lazy and just copy pasted. But lazy future me might uncomment without looking.